app/controllers/documents_controller.rb

```diff
@@ -1,5 +1,20 @@
 class DocumentsController < ApplicationController
-  before_action :require_intake
+  include AccessControllable
+  include FileResponseControllerHelper
```

Comment:
IMHO within the FileResponseControllerHelper, we need to start using `.service_url` from ActiveStorage as part of resolving a security issue we found a little while back. I consider it non-blocking for this pull request for now.

Using `.service_url` would mean redirecting to a S3 signed URL, which would ensure that if the client uploaded a HTML file, it renders on a different domain than ours, which separates its Javascript from our origin.